style.css

```diff
@@ -0,0 +1,100 @@
+#id-main-content {
+  box-shadow: 0 0 5px black;
+  width: 300px;
+  height: auto;
+  margin: auto;
+  background-image: url(/images/image_geometry_1.png);
```

Comment:
![image](https://user-images.githubusercontent.com/60631456/120654024-e691c600-c446-11eb-946e-871e8583e5d5.png)

- [ ] [OPTIONAL] If you deploy your project, there will be 3 errors to load the resources, the problem is the way you load the image, instead of /images/image_geometry_1.png, use ./images/image_geometry_1.png. If you are working in the live server of your browser, you only see one error, this is because Vscode helps you with it, but the deployment services will note. [LIVE DEMO OF YOUT PROJECT](https://rawcdn.githack.com/Marcraphael12/Marc-Raphael-Portofolio/92358436f6bcb65a2c3ca1326fdb13654d2a6d25/index.html)

Reply:
Thanks a lot!